concepts/recursion/about.md

```diff
@@ -0,0 +1,191 @@
+# About
+
+Recursion is a way to repeatedly execute code inside a function through the function calling itself.
+Functions that call themselves are know as _recursive_ functions.
+Recursion can be viewed as another way to loop/iterate. 
+And like looping, a Boolean expression or `True/False` test is used to know when to stop the recursive execution.
+_Unlike_ looping, recursion without termination in Python cannot not run infinitely.
+Values used in each function call are placed in their own frame on the Python interpreter stack.
+If the total amount of function calls takes up more space than the stack has room for, it will result in an error.
+
+## Looping vs Recursive Implementation
+
+Looping and recursion may _feel_ similar in that they are both iterative.
+However, they both _look_ differently, both at the code level and at the implementation level.
+Looping can take place within the same frame on the call stack.
+It usually manages this by updating one or more variables to maintain state as it iterates.
+This is an efficient implementation, but it can be somewhat cluttered at the code level.
+Recursion, instead of updating state variables, can pass updated values as arguments to the next call of the same function.
+This declutters the body of the function, but it is a less effficient implementation, as each call to the same function adds another frame to the stack.
+
+## Recursion: Why and Why Not?
+
+If there is risk of causing a stack error or overflow, why would anyone use a recursive strategy to solve a problem?
+There may be situations where a solution is more readable and/or easier to reason through when expressed through recursion than when expressed through looping.
+There may also be program constraints with using/mutating data, managing complexity, delegating responsibility, or organizing workloads.
+Problems that lend themselves to recursion include complex but repetitive problems that grow smaller over time, particularly [divide and conquer][divide and conquer] algorithms and [cumulative][cumulative] algorithms.
+However, due to Python's limit for how many frames are allowed on the stack, not all problems will benefit from a fully recursive strategy.
```

Comment:
```suggestion
```

Reply:
This was your addition, and I thought it was a good one, so I don't want to delete it..